tests/modules/programs/ncmpcpp/default.nix

```diff
@@ -0,0 +1,5 @@
+{
+  ncmpcpp-use-mpd-config = ./ncmpcpp-use-mpd-config.nix;
```

Comment:
This test should only happen for Darwin targets.

Reply:
I don't follow. In your [previous comment](https://github.com/rycee/home-manager/pull/1457#discussion_r479776480) you seem to be suggesting that the MPD configuration should _not_ be used if the target is Darwin. But this test checks that the MPD configuration _is_ used, if `programs.ncmpcpp.mpdMusicDir` isn't set directly. So shouldn't this test only happen for non-Darwin targets? Or even Linux-only targets?

Reply:
Sorry! Yeah, it should definitely be for Linux. The simplest way I imagine is to move the test to a different directory, like `tests/modules/programs/ncmpcpp-linux`, and add that in the Linux only section of `test/default.nix`.

Reply:
Right, thanks for the clarification and directions on how to test for Linux only! I'll try to make some time tonight to make the changes.